goja/EventAnalysis.cpp

```diff
@@ -195,6 +195,69 @@ EventType verify_type_of_coincidence(const Hit &h1, const  Hit &h2) {
 
 }
 
+std::tuple<bool, std::vector<Hit>> check_if_2plus1(const std::vector<Hit>& hits)
+{
+  double COMPTON_E_TH_PROMPT = atof(getenv("GOJA_COMPTON_E_TH_PROMPT"))*1e3;
+  
+  std::vector<Hit> triple_hits;
+
+  for (unsigned int i = 0; i < hits.size(); i++)
+  {
+    if (hits[i].edep <= COMPTON_E_TH_PROMPT)
+    {
+      triple_hits.push_back(hits[i]);
+    }
+  }
```

Comment:
A bit modern way would be
```suggestion
  for (auto hit&: hits )
  {
    if (hit.edep <= COMPTON_E_TH_PROMPT)
    {
      triple_hits.push_back(hit);
    }
  }
```

Reply:
Done